src/test/java/org/alfresco/repo/web/scripts/groups/GroupsTest.java

```diff
@@ -357,57 +356,6 @@ public void testGetGroup() throws Exception
     
     }
 
-	/**

```

Comment:
Any reason for removing the test

Reply:
Hi @michaelsuzukisagi it was a test added by me. After talking with Meenal, we realized the tests have to be implemented in the TAS RESTAPI project. 

So I left the test class as it was, (because the services tested by GroupsTest are not what we are fixing, they belong to some old api), and I'm implementing them in the other repository

Reply:
I prefer having more unit tests over integration tests.

Reply:
Added unit tests for this issue.